workflows-scripts/item-search-filter/README.md

```diff
@@ -0,0 +1,78 @@
+Steps to run JMeter script:
+Using GUI:
+Open .jmx file in JMeter IDE and make sure host and credentials are valid and hit the play button
+
+Using non-GUI:
+Navigate to Apache bin folder and run script.
+For example:
+Workspace/apache-jmeter-5.2.1/bin                                                                                                                                                                                                                            
+▶ sudo ./jmeter.sh -n -t /Users/vjavalkar/Workspace/FOLIO/perf-testing/workflows-scripts/item-search-filter/UserSearch.jmx -l /Users/vjavalkar/Workspace/FOLIO/perf-testing/workflows-scripts/item-search-filter/result.jtl 
+Password:
+Creating summariser <summary>
+Created the tree successfully using /Users/vjavalkar/Workspace/FOLIO/perf-testing/workflows-scripts/item-search-filter/UserSearch.jmx
+Starting standalone test @ Fri Apr 17 14:06:38 EDT 2020 (1587146798332)
+Waiting for possible Shutdown/StopTestNow/HeapDump/ThreadDump message on port 4445
+summary +     15 in 00:00:23 =    0.7/s Avg:  1482 Min:   160 Max:  1866 Err:     0 (0.00%) Active: 1 Started: 1 Finished: 0
+summary +     17 in 00:00:30 =    0.6/s Avg:  1765 Min:  1659 Max:  1890 Err:     0 (0.00%) Active: 1 Started: 1 Finished: 0
+summary =     32 in 00:00:53 =    0.6/s Avg:  1632 Min:   160 Max:  1890 Err:     0 (0.00%)
+summary +     17 in 00:00:30 =    0.6/s Avg:  1772 Min:  1669 Max:  1910 Err:     0 (0.00%) Active: 1 Started: 1 Finished: 0
+summary =     49 in 00:01:23 =    0.6/s Avg:  1681 Min:   160 Max:  1910 Err:     0 (0.00%)
+summary +     16 in 00:00:28 =    0.6/s Avg:  1775 Min:  1663 Max:  1936 Err:     0 (0.00%) Active: 1 Started: 1 Finished: 0
+summary =     65 in 00:01:51 =    0.6/s Avg:  1704 Min:   160 Max:  1936 Err:     0 (0.00%)
+summary +     17 in 00:00:30 =    0.6/s Avg:  1765 Min:  1662 Max:  1898 Err:     0 (0.00%) Active: 1 Started: 1 Finished: 0
+summary =     82 in 00:02:21 =    0.6/s Avg:  1716 Min:   160 Max:  1936 Err:     0 (0.00%)
+summary +     12 in 00:00:21 =    0.6/s Avg:  1775 Min:  1676 Max:  1950 Err:     0 (0.00%) Active: 0 Started: 1 Finished: 1
+summary =     94 in 00:02:43 =    0.6/s Avg:  1724 Min:   160 Max:  1950 Err:     0 (0.00%)
+Tidying up ...    @ Fri Apr 17 14:09:21 EDT 2020 (1587146961403)
+... end of run
+
+Workspace/apache-jmeter-5.2.1/bin                                                                                                                                                                                                                            
+▶ sudo ./jmeter.sh -n -t /Users/vjavalkar/Workspace/FOLIO/perf-testing/workflows-scripts/item-search-filter/InventoryInstanceSearch.jmx -l /Users/vjavalkar/Workspace/FOLIO/perf-testing/workflows-scripts/item-search-filter/result.jtl 
```

Comment:
How about removing your personal path from the command? E.x. we can change this path to the template
`[path_to_project]/perf-testing/workflows-scripts/item-search-filter/InventoryInstanceSearch.jmx`